framework/midi/internal/soundfontsprovider.h

```diff
@@ -38,7 +41,13 @@ class SoundFontsProvider : public ISoundFontsProvider
 public:
 
     std::vector<io::path> soundFontPathsForSynth(const SynthName& synthName) const override;
+    async::Notification soundFontPathsForSynthChanged(const SynthName& synth) const override;
+
     std::vector<io::path> soundFontPaths(SoundFontFormats formats) const override;
+
+private:
+
+    mutable std::map<SynthName, async::Notification > m_soundFontPathsForSynthChanged;
```

Comment:
Maybe it's better to add Map to the name.

Reply:
fixed